<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LabelPathDlg</class>
 <widget class="QDialog" name="LabelPathDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>227</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Point list picking</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="revertToolButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="statusTip">
        <string>Remove last entry</string>
       </property>
       <property name="whatsThis">
        <string>Remove last entry</string>
       </property>
       <property name="text">
        <string>remove last</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/CC/images/smallReset.png</normaloff>:/CC/images/smallReset.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="validToolButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="statusTip">
        <string>Convert list to new cloud (and close dialog)</string>
       </property>
       <property name="whatsThis">
        <string>Convert list to new cloud (and close dialog)</string>
       </property>
       <property name="text">
        <string>to cloud</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/CC/images/smallValidate.png</normaloff>:/CC/images/smallValidate.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="cancelToolButton">
       <property name="toolTip">
        <string>Close dialog (list will be lost)</string>
       </property>
       <property name="statusTip">
        <string>Close dialog (list will be lost)</string>
       </property>
       <property name="text">
        <string>stop</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/CC/images/smallCancel.png</normaloff>:/CC/images/smallCancel.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>count</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="countLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string notr="true">0</string>
       </property>
       <property name="maxLength">
        <number>255</number>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>115</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>20</number>
     </attribute>
     <column>
      <property name="text">
       <string>路径点序号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">X</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">Y</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">Z</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>marker size</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="markerSizeSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>99</number>
       </property>
       <property name="value">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="showGlobalCoordsCheckBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Show global coordinates (instead of shifted ones)</string>
       </property>
       <property name="text">
        <string>show global coordinates</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
